### ISSUE TITLE
How to train when I set batch_normalize=0？

### ISSUE THREAD

**Issue by yeyewen** [AlexeyAB/darknet#4135]
@AlexeyAB  My work needs to train a model ， all the convolution with batch_normalize=0. I set
 batch_normalize=0 in my cfg. But it can't be trained,when I sstarted training, the loss always tend to be NAN . I tried
```
policy=sgdr
sgdr_cycle=1000
sgdr_mult=2
steps=65000,80000,88000
scales=0.1,0.1,0.1
```
and
```
policy=steps
steps=65000,80000,88000
scales=.1,.1,.1
```
So,can I train the model without batch_normalization?

**Comment by yeyewen** [comment]
I changed learning_rate from 0.001 to 0.0001，and it was helpful.
